### ISSUE TITLE
TypeError: NetworkError when attempting to fetch resource

### ISSUE THREAD

**Issue by diegobersanetti** [myfreeweb/transmitter#32]
Hello,
upon installation and configuration (double checked server, path, user and password already) I am facing this error, showing up in the pop-up:

TypeError: NetworkError when attempting to fetch resource

Clicking on the Transmission icon correctly redirects me to the server.

I don't know is this can help (I am no JS developer), but it seems related to the fetch API in Firefox and/or polyfill

I run Firefox 82.0.3 on Ubuntu 20.04.1

**Comment by valpackett** [comment]
Well, somehow the connection is failing. You can open the [extension debugger](https://extensionworkshop.com/documentation/develop/debugging/) and look in the Network tab.

**Comment by diegobersanetti** [comment]
I looked and it did not show anything, as it did not connect at all.
So I tried and re-typed the server name (exactly the same, though); this fixed the issue, as there was some formatting problem. Actually it could have been, as I already used the extension quite some time ago but disabled it as my server was down, and only recently came back online (with the same configuration, so I did not think about changing the server name).

Anyway, issue fixed, thanks for the suggestion!
